WordPress/src/main/java/org/wordpress/android/ui/ActionableEmptyView.kt

```diff
@@ -137,6 +137,13 @@ class ActionableEmptyView : LinearLayout {
         layout.layoutParams = params
     }
 
+    /**
```

Comment:
@jd-alexander I guess we'll need to remove this? 😉

Reply:
Good idea. Modified in f1d6f75